modules/op-release-notes-1-8.adoc

```diff
@@ -12,14 +12,263 @@ With this update, {pipelines-title} General Availability (GA) 1.8 is available o
 
 In addition to the fixes and stability improvements, the following sections highlight what is new in {pipelines-title} 1.8.
 
+[id="pipelines-new-features-1-8_{context}"]
+=== Pipelines
+
 :FeatureName: Running {pipelines-title} on ARM hardware
-[id="running-pipelines-on-arm-hardware-1-8_{context}"]
-=== {FeatureName}
+* {FeatureName} With this update, you can run {pipelines-title} GA 1.8 and later on an {product-title} cluster that is running on ARM hardware. This includes support for `ClusterTask` resources and the `tkn` CLI tool.
++
+include::snippets/technology-preview.adoc[]
+// Shubham Minglani
+
+* With this update, as a developer, you can add your Git repository to {pac} by using the webhook mechanism. This way, you can use {pac} even when using GitHub Apps is not feasible. This capability includes support for repositories on GitHub, GitLab, and BitBucket.
+// (link:https://issues.redhat.com/browse/SRVKP-2137[SRVKP-2137 Pipeline as code - CLI add repository via webhooks])
+// Shivam Mukhade
+
+* Previously, as a developer, you could have only one pipeline run in your `.tekton` directory for each Git event, such as a pull request. With this update, you can have all the pipeline runs in your `.tekton` directory match specified events. The web console displays the status and reports of the runs. If you are using WebVCS, the reports include comments.
+// (link:https://issues.redhat.com/browse/SRVKP-1781[SRVKP-1781 Multiple Pipelines runs match on events])
+// Chmouel Boudjnah
+
+* With this update, as a developer, you get a notification from the `tkn-pac` CLI tool if you try to add a duplicate repository to a {pac} run. When you enter `tkn pac repo create`, each repository must have a unique URL. This notification also helps prevent hijacking exploits.
```

Comment:
```suggestion
* With this update, as a developer, you get a notification from the `tkn-pac` CLI tool if you try to add a duplicate repository to a {pac} run. When you enter `tkn pac create repository`, each repository must have a unique URL. This notification also helps prevent hijacking exploits.
```

Reply:
this is Pipelines as Code feature shall we move it to Pipelines as Code section?

Reply:
@sm43 Are you sure about the new syntax you're proposing. We have it documented differently here: https://github.com/openshift/openshift-docs/blob/main/modules/op-pipelines-as-code-command-reference.adoc#repository

Reply:
Moved this to the PAC section.

Reply:
Removed the following release note as a duplicate:
* This update adds a new webhook validator to ensure only one {pac} repository is created by one Git repository URL.
// no link